packages/opentelemetry-sdk-node/README.md

```diff
@@ -0,0 +1,119 @@
+# OpenTelemetry SDK for NodeJS
+
+[![Gitter chat][gitter-image]][gitter-url]
+[![NPM Published Version][npm-img]][npm-url]
+[![dependencies][dependencies-image]][dependencies-url]
+[![devDependencies][devDependencies-image]][devDependencies-url]
+[![Apache License][license-image]][license-image]
+
+@ToDo
+This package provides...
+
+## Quick Start
+
+@ToDo
+To get started you need to ...
+
+### Installation
+
+```sh
+$ # Install the SDK
+$ npm install @opentelemetry/sdk-node
+
+$ # Install exporters and plugins
+$ npm install \
+    @opentelemetry/exporter-jaeger \ # add tracing exporters as needed
+    @opentelemetry/exporter-prometheus # add metrics exporters as needed
+    @opentelemetry/plugin-http # add plugins as needed
+```
+
+> Note: this example is for node.js. See [examples/tracer-web](https://github.com/open-telemetry/opentelemetry-js/tree/master/examples/tracer-web) for a browser example.
+
+### Initialize the SDK
+
+Before any other module in your application is loaded, you must initialize the SDK.
+If you fail to initialize the SDK or initialize it too late, no-op implementations will be provided to any library which acquires a tracer or meter from the API.
+This example uses Jaeger and Prometheus, but exporters exist for [other tracing backends][other-tracing-backends].
+
+```javascript
+const opentelemetry = require("@opentelemetry/sdk-node");
+const { JaegerExporter } = require("@opentelemetry/exporter-jaeger");
+const { PrometheusExporter } = require("@opentelemetry/exporter-prometheus");
+
+const jaegerExporter = new JaegerExporter({
+  serviceName: 'my-service',
+});
+const prometheusExporter = new PrometheusExporter({ startServer: true });
+
+const sdk = new opentelemetry.NodeSDK({
+  // Optional - if omitted, the tracing SDK will not be initialized
+  traceExporter: jaegerExporter,
+  // Optional - If omitted, the metrics SDK will not be initialized
+  metricExporter: prometheusExporter,
+
+  // See the Configuration section below for additional  configuration options
+});
+
+// You can optionally detect resources asynchronously from the environment.
+// Detected resources are merged with the resources provided in the SDK configuration.
+sdk
+  .autoDetectResources()
```

Comment:
I believe resource detection should be integral part of the SDK, I should be available implicitly in the SDK. No extra call required from the users to register it. WDYT?

Reply:
Resource detection may do some async work so we need some way for users to wait for it. If it is done at construction there is no way to wait for it.

Reply:
Would having a `detectResources` flag on `NodeSDKConfiguration` be a reasonable compromise? If set to true, start would block until they resolved, but it would save the additional API call. I'm neutral as to whether it'd default to true or false.